src/lxml/parser.pxi

```diff
@@ -1615,6 +1615,8 @@ cdef class HTMLParser(_FeedParser):
     - strip_cdata        - replace CDATA sections by normal text content (default: True)
     - compact            - save memory for short text content (default: True)
     - default_doctype    - add a default doctype even if it is not found in the HTML (default: True)
+    - collect_ids        - Set this to False to avoid ever-growing id cache in
+                           long-running processes (at a bit of a performance cost)
```

Comment:
That reads a bit biased. The downside is that you'd loose ID lookups, that should be mentioned instead.